tv_apps/smart-home/js/filter_manager.js

```diff
@@ -138,8 +138,26 @@
       }
       this._isFilterChanging = true;
       this._cardListElem.setAttribute('smart-bubbles-direction', 'down');
-      this._smartBubblesElem.play(
-        document.querySelectorAll('#card-list > .card > .app-button'));
+
+      // Notice that document.querySelectorAll returns elements based on the
```

Comment:
Oh, thanks for addressing this problem.
I would consider it as a mismatch between how `scrollable.js` works and DOM structure itself.
And since the logic that really cause the order mismatch with its real DOM tree order is inside `scrollable.js`, I think it would be better to move the code below into `scrollable.js` as a function called `sortNodes`, and then call it here.
How do you think?